docs/DeveloperGuide.md

```diff
@@ -6,8 +6,248 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+## Design
 
+### Add Housekeeper Command
+
+The objective of the `AddHousekeeperCommand` is to take in user input and spilt it into two parts which is the name and age of a Housekeeper. These details make up parts of the Housekeeper profile. With the name and age derived, this information will be added into a new Housekeeper object, which will be recorded into the list of housekeeper. The class diagram below depicts how the `AddHousekeeperCommand` interacts with other classes.
+
+![class](team/falicia_addHousekeeperCommand/classAddHousekeeper.jpg)
```

Comment:
I think this diagram is too complicated for what it claims to do, since it shows lots of internal details when its goal is to show how `AddHousekeeperCommand` interacts with other classes. Consider focusing on important associations and only leaving in important attributes? (The same can be said for the `AddItemCommand`'s class diagram.)